legacy/genesis_theta_local.py

```diff
@@ -61,19 +61,25 @@
 
 LOCAL_PUBKEY = "qwiUMxz3llsy45fPvM0a8+XQeAJLvrX3QAEJmRMEEoU="
 LOCAL_ADDRESS = "D5AB5E458FD9F9964EF50A80451B6F3922E6A4AA"
-LOCAL_CONS_ADDRESS = "cosmosvalcons16k44u3v0m8uevnh4p2qy2xm08y3wdf92xsc3ve" 
+LOCAL_CONS_ADDRESS = "cosmosvalcons16k44u3v0m8uevnh4p2qy2xm08y3wdf92xsc3ve"
 
 PREPROCESSED_FILE = GENESIS_ARCHIVE + ".preprocessed.json"
 
 shutil.copy2(GENESIS_ARCHIVE, PREPROCESSED_FILE)
 
-subprocess.call(["sed", "-i", ".bak", "s%" + COINBASE_SELF_DELEGATION_ADDR    + "%" + LOCAL_SELF_DELEGATION_ADDR   + "%g",PREPROCESSED_FILE])
-subprocess.call(["sed", "-i", ".bak", "s%" + COINBASE_SELF_DELEGATION_PUBKEY  + "%" + LOCAL_SELF_DELEGATION_PUBKEY + "%g",PREPROCESSED_FILE])
-subprocess.call(["sed", "-i", ".bak", "s%" + COINBASE_VALIDATOR_OP_ADDRESS    + "%" + LOCAL_VALIDATOR_OP_ADDRESS   + "%g",PREPROCESSED_FILE])
+subprocess.call(["sed", "-i", ".bak", "s%" + COINBASE_SELF_DELEGATION_ADDR +
```

Comment:
should we remove the legacy folder @dasanchez? not sure what the value in keeping it is?

Reply:
I'd moved the old examples to a legacy folder just to have them as reference, but yes, I'd rather delete them instead.